### DISCUSSION TITLE
Why display alias went removed?

### DISCUSSION THREAD

**Discussion by raulvictorrosa** [chakra-ui/chakra-ui#6223] [Q&A]
Before version 2 we have the option to use display with the alias **d**, I would like to know why it got removed?

**Comment by aroyan** [comment]
They don't give an explanation about this, just said that is removed. You can check on here [Changelog](https://chakra-ui.com/changelog#chakra-uireact-v2)
![image](https://user-images.githubusercontent.com/43630681/175489824-9da61698-97d8-479f-b96f-1edd1b7aa550.png)
